couchbase-operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: couchbase-operator
 description: A Helm chart to deploy the Couchbase Autonomous Operator for easily deploying, managing, and maintaining Couchbase Clusters. Couchbase Server is a NoSQL document database with a distributed architecture for performance, scalability, and availability. It enables developers to build applications easier and faster by leveraging the power of SQL with the flexibility of JSON.
-version: 2.0.1
+version: 2.0.2
```

Comment:
I'd like to leave this as 2.0.1 to avoid version confusion in places like helmhub.
Going forward we will probably include a build number instead a way to indicate patches...ie 2.0.1+68e229c.  (this would be applied post-commit, after pr is merged)
https://semver.org/#spec-item-10